lib/Gocdb_Services/Validate.php

```diff
@@ -36,6 +36,22 @@ public function validate($Object_Name, $Field_Name, $Field_Value)
         //Check the length of the field value, if too long, throws exception
         $this->checkFieldLength($Object_Name, $Field_Name, $Field_Value);
 
+        //Check the Type of the field value, where this is defined
+        $type = $this->Get_Field_value($Object_Name, $Field_Name, 'ftype');
+        if (count($type) != 0) {
```

Comment:
Does `Get_Field_value` returns an array?

Reply:
I haven't checked, but I suspect it returns an array with one element which is the value of that field, or an empty array if it couldn't find anything. This snippet of code was copied from elsewhere in the validation code.